### PR TITLE
Initial changes

### PR DESCRIPTION
**Pull request by shashisk97**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

JDDO/LADO dynamic list issue fix

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by sunilkumar-cts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/DirectionOrderService.java [hunk 1]
```diff
@@ -16,18 +18,35 @@ public DirectionOrderService(PilotCourtService pilotCourtService) {
     }
 
     public HearingCourt getHearingCourt(CCDCase ccdCase) {
-        if (StringUtils.isBlank(ccdCase.getHearingCourt())
-            || ccdCase.getHearingCourt().equals(PilotCourtService.OTHER_COURT_ID)) {
+        if (ccdCase.getHearingCourt() instanceof LinkedHashMap) {
+            LinkedHashMap<String, Object> hearingCourt = (LinkedHashMap) ccdCase.getHearingCourt();
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/DirectionOrderService.java:22]: Do a null and containsKey check please.

File: ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/DirectionOrderMapper.java [hunk 2]
```diff
@@ -41,7 +41,8 @@ public void from(CCDCase ccdCase, Claim.ClaimBuilder claimBuilder) {
         Optional.ofNullable(ccdCase.getEstimatedHearingDuration()).ifPresent(estimatedDuration ->
             builder.estimatedHearingDuration(HearingDurationType.valueOf(estimatedDuration.name())));
 
-        Optional.ofNullable(ccdCase.getHearingCourt()).ifPresent(builder::hearingCourt);
+        Optional.ofNullable(ccdCase.getHearingCourt()).ifPresent(hearingCourt ->
+            builder.hearingCourt(String.valueOf(hearingCourt)));
```
Comment (AlokDatta) [inline ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/DirectionOrderMapper.java:45]: Please do a regression check on this event.

**Review by AlokDatta — APPROVED** [review]

**Comment by shashisk97** [comment]
Closing and creating new one.

**Comment by shashisk97** [comment]
2324 new pr

**Pull request closed without merging**